src/components/Article.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({ title, date, text }) => (
+  <>
+    <h1>{title}</h1>
+    <span>{date}</span>
+    <p>{text}</p>
+  </>
+);
+
+Article.propTypes = {
+  // eslint-disable-next-line react/require-default-props
+  title: PropTypes.string,
```

Comment:
You should use isRequired here or set defaultProps for non required fields

Reply:
Added isRequired